app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -263,30 +256,41 @@ fun RunHikeBottomSheet(
 
             DetailRow(
                 label = stringResource(R.string.run_hike_screen_label_current_elevation),
-                value = "50m")
+                // TODO hardcoded to 50m for now
+                value =
+                    stringResource(R.string.run_hike_screen_value_format_current_elevation)
+                        .format(50))
             DetailRow(
                 label = stringResource(R.string.run_hike_screen_label_elevation_gain),
-                value = "${elevationGainString}m")
+                value =
+                    stringResource(R.string.run_hike_screen_value_format_elevation_gain)
+                        .format(hikeRoute.elevationGain.roundToInt()))
             DetailRow(
                 label = stringResource(R.string.run_hike_screen_label_estimated_time),
                 value =
-                    if (hikeRoute.estimatedTime / 60 < 1) "${minuteString}min"
-                    else "${hourString}h${minuteString}")
+                    if (hikeRoute.estimatedTime / 60 < 1)
+                        stringResource(R.string.run_hike_screen_value_format_estimated_time_minutes)
+                            .format((hikeRoute.estimatedTime % 60).roundToInt())
+                    else
+                        stringResource(
+                                R.string
+                                    .run_hike_screen_value_format_estimated_time_hours_and_minutes)
+                            .format(
+                                (hikeRoute.estimatedTime / 60).toInt(),
+                                (hikeRoute.estimatedTime % 60).roundToInt()))
```

Comment:
Nitpick: you actually don't need to call `format` manually, you can just pass the arguments directly to `stringResource`, it calls `format` under the hood.

```suggestion
                        stringResource(
                                R.string
                                    .run_hike_screen_value_format_estimated_time_hours_and_minutes, (hikeRoute.estimatedTime / 60).toInt(),
                                (hikeRoute.estimatedTime % 60).roundToInt()))
```

Reply:
done